src/common/workspace-store.ts

```diff
@@ -130,6 +130,10 @@ export class Workspace implements WorkspaceModel, WorkspaceState {
     Object.assign(this, state);
   }
 
+  @action updateModel(model: WorkspaceModel) {
```

Comment:
Should this be private/protected?

Reply:
We can make this a unexported symbol if you would like